core/logger/export_test.go

```diff
@@ -0,0 +1,39 @@
+package logger
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func (el *Logger) ErrorWithoutFileRoll(message string, extra ...interface{}) {
+	el.errorWithoutFileRoll(message, extra)
+}
+
+func TestRedirectStderr(t *testing.T) {
+	file, _ := newFile("", "logs", "log")
+	err := redirectStderr(file)
+	assert.Nil(t, err)
+}
+
+func TestRedirectStderrWithNilFile(t *testing.T) {
+	err := redirectStderr(nil)
+	assert.NotNil(t, err)
+}
+
+func TestRollFiles(t *testing.T) {
+	t.Parallel()
```

Comment:
no testing in export_test.go, here you only export internal methods.

Reply:
Changed.